.travis.yml

```diff
@@ -14,9 +14,10 @@ install:
   - source activate env_name
   - if [ ${PYTHON_VERSION} = "3.7" ]; then pip install sphinx==1.2.2 "docutils<0.14"; fi
   - pip install coveralls
+  - pip install anndata
```

Comment:
Would it be possible to set this up as an optional build for the build matrix? (e.g., in `env` put `- PYTHON_VERSION=3.6 WITH_ANNDATA=True`, and then use a conditional here for install?

Reply:
wouldn't this make it fail for the other versions?

Reply:
Ah, good point. Since this is an optional dependency, it would be good to setup the tests with a skip decorator -- sorry for not catching that on the initial review. See [here](https://github.com/biocore/biom-format/blob/master/biom/tests/test_table.py#L584) as an example of how it could be done.

Reply:
please let me know if you like the new solution.